dnase/callsnpsMerge.sh

```diff
@@ -163,6 +164,41 @@ bcftools index ${sampleOutdir}/${name}.${mappedgenome}.filtered.vcf.gz
 tabix -p vcf ${sampleOutdir}/${name}.${mappedgenome}.filtered.vcf.gz
 rm -f ${fltvcffiles}
 
+echo
+echo "Run Delly"
```

Comment:
add comment stating DELLY is not easily parallelizeable, hence why it runs in callsnpsMerge